### ISSUE TITLE
Error messages are out of vim way

### ISSUE THREAD

**Issue by sudo-nice** [vifm/vifm#233]
![screenshot](http://i.imgur.com/AzURY5b.png)

I think, the error messages have two disadvantages now:
1. They are mc-like style, rather then vim-like.
2. They demand an extra keypress to continue the work.

Since we have the command line always enabled and it can show error messages, wouldn't it be more straight to use the command line for error messages? This way we can learn about errors but continue the work without any breaks. And this is exactly the way vim show errors.

What do you think?

**Comment by xaizek** [comment]
There are error messages on command-line, but the way errors are displayed depends on their origin. If it's a direct result of a command-line command or error that is likely to happen, then it will be on the command-line. But when it's something we don't normally expect or it might follow from changes of file-system by another application, it appears as a dialog to make sure user won't miss the error and will not by accident do anything bad (e.g., one could press <kbd>lDD</kbd> and if we don't stop on error after <kbd>l</kbd>, wrong file might get deleted). In Vim it's much harder to mess up significant part of your file system with a single key press.

So basically errors that are hard to predict are intentionally nagging to be hard to ignore.

**Comment by sudo-nice** [comment]
Got it, thank you.
